porcelain/wallet.go

```diff
@@ -2,15 +2,22 @@ package porcelain
 
 import (
 	"context"
+	"gx/ipfs/QmVmDhyTTUcQXFD1rRQ64fGLMSAoaQvNH3hwuaCFAPq2hy/errors"
 
 	"github.com/filecoin-project/go-filecoin/actor"
 	"github.com/filecoin-project/go-filecoin/address"
 	"github.com/filecoin-project/go-filecoin/state"
 	"github.com/filecoin-project/go-filecoin/types"
 )
 
+// ErrNoDefaultFromAddress is returned when a default wallet address couldn't be determined (eg, there are zero addresses in the wallet).
+var ErrNoDefaultFromAddress = errors.New("unable to determine a default wallet address")
+
 type walletPlumbing interface {
 	ActorGet(ctx context.Context, addr address.Address) (*actor.Actor, error)
+	ConfigGet(dottedPath string) (interface{}, error)
```

Comment:
This mixes the dependencies for these two calls together. They should be separate so we can what what each depends on. Would you please break this into two interfaces, wbPlumbing which has actorget for walletbalance and wdaPlumbing which has configget/set/walletaddresses for walletdefaultadddress? then ensure that walletdefaultaddress takes wdaPlumbing.

Reply:
thanks,I've done it